structs_test.go

```diff
@@ -296,6 +296,33 @@ func TestMap_Anonymous(t *testing.T) {
 	}
 }
 
+func TestMap_Flatnested(t *testing.T) {
+	type A struct {
+		Name string
+	}
+	a := A{Name: "example"}
+
+	type B struct {
+		A `structs:",flatnested"`
+		C int
```

Comment:
Do you prefer a panic in case of overwrite field to prevent any possible mistake? or simply an overwrite (based on the the order of fields returned by reflect package)?